### DISCUSSION TITLE
Converting to Open-Shell starting from using Class Shell now in Win 10

### DISCUSSION THREAD

**Discussion by billmartz** [Open-Shell/Open-Shell-Menu#1865] [Q&A] [answered]
I have been using Classic Shell for years, and I have a many layered start menu that I want to have after installing Open-Shell. To install Open-Shell do I uninstall Classic Shell first? What happens to all the shortcuts that are in the menus? Do I have to make screen-shots of the menu system and then repopulate with new shortcuts manually? Can this be done before I switch to Win 11?

**Comment by ge0rdi** [comment]
You don't need to uninstall `Classic Shell`.
`Open-Shell` will replace it and use its settings/data.

Though for sure create some backups, screenshots.

**Comment by billmartz** [comment, marked as answer]
Open-Shell takes right over for Classic Shell. You don't have to do a thing but install it.

**Comment by billmartz** [comment]
Thanks. You're right.

Bill
